tests/resource_response_test.py

```diff
@@ -87,6 +88,22 @@ def test_cancelled_returns_false_if_called_before_cancel(self):
         self.assertFalse(self.future.cancelled())
 
 
+class HandleResponseErrorsTest(unittest.TestCase):
+
+    def test_pass_response_and_request(self):
+        request = Mock()
+        response = Mock(text='{"foo": "bar"}')
+        error = Mock(
+            args=('400 Bad Request',), response=response, request=request
+        )
+        try:
+            handle_response_errors(error)
+        except HTTPError as e:
+            self.assertEqual(e.json, {'foo': 'bar'})
```

Comment:
Ah, that was from previous diff where I had added that as a property to exception. Fixed <3